test/functional/rpc_dumptxoutset.py

```diff
@@ -18,16 +19,12 @@ def set_test_params(self):
         self.setup_clean_chain = True
         self.num_nodes = 1
 
-    def run_test(self):
-        """Test a trivial usage of the dumptxoutset RPC command."""
+    def test_dump_file(self, filename, is_human_readable, expected_digest):
+
```

Comment:
```suggestion
```
Feel free to ignore: I'd remove this blank line

Reply:
Done. Suggestion accepted.